doc/recipe/preprocessor.rst

```diff
@@ -65,7 +65,7 @@ the preprocessor structure see the :ref:`Preprocessors`.
 
 Each of the preprocessor operations is written in a dedicated python module and
 all of them receive and return an Iris `cube
-<https://scitools.org.uk/iris/docs/v2.0/iris/iris/cube.html>`_ , working
+<https://scitools-iris.readthedocs.io/en/latest/generated/api/iris/cube.html>`_ , working
```

Comment:
You can just use interphinx to directly link to iris functions and classes in most places, that avoids this problem recurring if the documentation is moved again. E.g. this link can be replace by
```
:obj:`iris.cube.Cube`
```

Reply:
Great to know! Thanks, Bouwe!